### PR TITLE
Install torchserve from source

### PR DESCRIPTION
**Pull request by mycpuorg**
This addresses issue #312 updates README
with instructions to install from local path.
It builds and installs the pip whl file

## Description

Please include a summary of the feature or issue being fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #(issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] This change requires a documentation update

## Feature/Issue validation/testing

Please describe the tests [UT/IT] that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced. 
Please also list any relevant details for your test configuration.

- [x] Test A
Installed with `pip install .`

- UT/IT execution results

- Logs

## Checklist:

- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] New and existing unit tests pass locally with these changes?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,30 +106,24 @@ If you plan to develop with TorchServe and change some of the source code, you m
     sudo apt-get install openjdk-11-jdk
     ```
 
-1. Install dependencies
-
-    ```bash
-    pip install psutil future
```
Comment (vdantu) [inline README.md:112]: You might need `psutil` to build. [code reference](https://github.com/pytorch/serve/blob/master/ts/metrics/process_memory_metric.py#L10) . I think the above code reference will be invoked when running torch-serve.

File: README.md [hunk 2]
```diff
@@ -106,30 +106,24 @@ If you plan to develop with TorchServe and change some of the source code, you m
...
     ```bash
     git clone https://github.com/pytorch/serve
     cd serve
     ```
 
-1. Make your changes executable
+1. Install from local path
 
     ```bash
-    pip install -e .
```
Comment (vdantu) [inline README.md:125]: Should we just add the manual build step here
```
python setup.py build
```
and then
```
pip install . # or pip install -U -e .
```
the `python setup.py build` should invoke [build in setup.py](https://github.com/pytorch/serve/blob/master/setup.py#L91) which should technically build the frontend and plugins and everything else :) .

Comment (mycpuorg) [reply README.md:125]: sounds like we should add this to scripts in PR #322 ?

File: README.md (hunk 1 above)
Comment (mycpuorg) [reply README.md:112]: This is added as a dep inside setup.py. This way the conda binaries which are built out of pip won't have to install the deps separately.

**Comment by mycpuorg** [comment]
I'm leaning towards abandoning this in favor of PR #322

**Pull request closed without merging**